build.gradle.kts

```diff
@@ -107,6 +107,8 @@ allprojects {
         testImplementation("com.nhaarman.mockitokotlin2:mockito-kotlin:_")
         // Test runtime: Junit engine
         testRuntimeOnly(junit("engine"))
+        // Test runtime: Alchemist cognitive agents
+        testRuntimeOnly("it.unibo.alchemist:alchemist-cognitive-agents:_")
```

Comment:
Ah, I see. Please change as follows:
`testRuntimeOnly(alchemist("cognitive-agents"))`

otherwise, it will refer to a published artifact instead of accessing the subproject. Also, this must **not** be under `allprojects`, it's a dependency of the parent project.